<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PedidoNuevo</class>
 <widget class="QDialog" name="PedidoNuevo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Proveedor</string>
     </property>
     <property name="buddy">
      <cstring>lineEditIdProveedor</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditIdProveedor">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEditProveedor"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>NÂº. documento</string>
     </property>
     <property name="buddy">
      <cstring>lineEditDocumento</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEditDocumento"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Fecha documento</string>
     </property>
     <property name="buddy">
      <cstring>dateEditDocumento</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDateEdit" name="dateEditDocumento">
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAceptar">
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditIdProveedor</tabstop>
  <tabstop>lineEditProveedor</tabstop>
  <tabstop>lineEditDocumento</tabstop>
  <tabstop>dateEditDocumento</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonAceptar</sender>
   <signal>clicked()</signal>
   <receiver>PedidoNuevo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancelar</sender>
   <signal>clicked()</signal>
   <receiver>PedidoNuevo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
